src/main/java/com/epam/ta/reportportal/core/imprt/impl/junit/XunitImportStrategy.java

```diff
@@ -106,6 +110,11 @@ private String processZipFile(File zip, String projectId, String userName) throw
 			finishLaunch(launchId, projectId, userName, processResults(futures));
 			return launchId;
 		} catch (InterruptedException | ExecutionException | TimeoutException | IllegalArgumentException e) {
+			if (savedLaunchId != null) {
+				Launch launch = launchRepository.findOne(savedLaunchId);
+				launch.setStartTime(new Date());
+				launchRepository.partialUpdate(launch);
```

Comment:
partial update does not make sense here since 'launch' object has all the fields populated from by .findOne method. 
Please consider the following example that update date only:
```java
Launch launch = new Launch;
launch.setStatistics(null);
//set all non-null fields to null

launch.setDate(Calendar.getInstance().getTime());
repo.partialUpdate(launch);
```